indico/util/caching.py

```diff
@@ -30,7 +30,8 @@ def make_hashable(obj):
         return frozenset((k, make_hashable(v)) for k, v in obj.iteritems())
     elif hasattr(obj, 'getId') and obj.getId.__self__ is not None:
         # getId of AvatarUserWrapper would access a cached property, we can't have that here
-        id_ = obj.getId() if obj.__class__.__name__ != 'AvatarUserWrapper' else obj.id
+        # don't call Conference.getId() so we get the getId() access warning for other places
+        id_ = obj.getId() if obj.__class__.__name__ not in ('AvatarUserWrapper', 'Conference') else obj.id
```

Comment:
Sure, we can remove them and then get rid of this ;) But since they are still used right now I left this so I don't get the warning about accessing the legacy method from something that's always called